bin/tronfig

```diff
@@ -124,37 +124,31 @@ def validate_dir(path):
             shutil.rmtree(manifest_dir)
 
 
-def get_config_input(namespace, source=None):
-    if source is None:
-        source = namespace
-        namespace = None
-
+def get_config_input(source):
     if source == '-':
         source_io = sys.stdin
-        if not namespace:
-            namespace = schema.MASTER_NAMESPACE
+        namespace = schema.MASTER_NAMESPACE
```

Comment:
It looks like we lost the ability to read a specific namespace (non MASTER from stdin)

We still need this functionality, please make sure there is a namespace argument (-p) sent in so we can still use our existing tools